src/KubeLibrary/KubeLibrary.py

```diff
@@ -78,13 +78,13 @@ def reload_config(self, kube_config=None, context=None, incluster=False, cert_va
                 config.load_kube_config(kube_config, context)
             except TypeError:
                 logger.error('Neither KUBECONFIG nor ~/.kube/config available.')
-
         self._add_api('v1', client.CoreV1Api)
         self._add_api('extensionsv1beta1', client.ExtensionsV1beta1Api)
         self._add_api('batchv1', client.BatchV1Api)
         self._add_api('appsv1', client.AppsV1Api)
         self._add_api('batchv1_beta1', client.BatchV1beta1Api)
         self._add_api('custom_object', client.CustomObjectsApi)
+        self._add_api('rbac_authv1_api', client.RbacAuthorizationV1Api)
```

Comment:
Please add the new api in https://github.com/devopsspiral/KubeLibrary/blob/6215736fbc092a51ac194f4dcefaa8296811cb5b/test/test_KubeLibrary.py#L91 to cover it with tests.

Reply:
Done.